app/code/Magento/Checkout/view/frontend/layout/default.xml

```diff
@@ -58,6 +58,7 @@
                 </arguments>
                 <container name="minicart.addons" label="Mini-cart promotion block"/>
             </block>
+            <block class="Magento\Checkout\Block\Cart\Sidebar" name="minicart_updated_at" after="minicart" template="Magento_Checkout::cart/minicart_updated_at.phtml" cacheable="false"/>
```

Comment:
This will prevent full page caching for all pages. We need to have another solution

Reply:
Just minicart_updated_at.phtml won't have cache

Reply:
Hummm now I see your point, I'll refactor it.